src/styles/utils/reset.scss

```diff
@@ -0,0 +1,8 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+}
```

Comment:
In my opinion, you should move reset styles from `*` to `body`, and font styles from `body` to `html` or `:root`